modules/ducktests/tests/docker/Dockerfile

```diff
@@ -42,28 +42,23 @@ COPY ./ssh-config /root/.ssh/config
 RUN ssh-keygen -m PEM -q -t rsa -N '' -f /root/.ssh/id_rsa && cp -f /root/.ssh/id_rsa.pub /root/.ssh/authorized_keys
 RUN echo 'PermitUserEnvironment yes' >> /etc/ssh/sshd_config
 
-RUN chmod a+wr /opt
-
 ARG APACHE_MIRROR="https://apache-mirror.rbc.ru/pub/apache/"
 ARG APACHE_ARCHIVE="https://archive.apache.org/dist/"
 
 # Install binary test dependencies.
-ARG IGNITE_VERSION="2.8.0"
-ARG IGNITE_NAME="ignite-$IGNITE_VERSION"
-ARG IGNITE_RELEASE_NAME="apache-ignite-$IGNITE_VERSION-bin"
+RUN cd /opt && curl -O $APACHE_ARCHIVE/ignite/2.7.6/apache-ignite-2.7.6-bin.zip && unzip apache-ignite-2.7.6-bin.zip && mv /opt/apache-ignite-2.7.6-bin /opt/ignite-2.7.6
+RUN cd /opt && curl -O $APACHE_ARCHIVE/ignite/2.8.0/apache-ignite-2.8.0-bin.zip && unzip apache-ignite-2.8.0-bin.zip && mv /opt/apache-ignite-2.8.0-bin /opt/ignite-2.8.0
+RUN cd /opt && curl -O $APACHE_ARCHIVE/ignite/2.8.1/apache-ignite-2.8.1-bin.zip && unzip apache-ignite-2.8.1-bin.zip && mv /opt/apache-ignite-2.8.1-bin /opt/ignite-2.8.1
 
-ADD $APACHE_ARCHIVE/ignite/$IGNITE_VERSION/$IGNITE_RELEASE_NAME.zip /opt/
-RUN cd /opt && unzip $IGNITE_RELEASE_NAME.zip && rm $IGNITE_RELEASE_NAME.zip
-RUN mv /opt/$IGNITE_RELEASE_NAME /opt/$IGNITE_NAME
-RUN chmod a+wr /opt/$IGNITE_NAME -R
+RUN rm /opt/apache-ignite-*-bin.zip
+RUN chmod a+wr /opt -R
```

Comment:
That's just create two additional full-archive copy layers which increase image size [1] and may slow down file deletion operations due to layered nature of docker image [2]. 
```
$ docker images
REPOSITORY                TAG                 IMAGE ID            CREATED             SIZE
ducker-ignite-openjdk-8   latest              eb0bc05f45f0        10 minutes ago      4.31GB

ducker@ducker01:/$ sudo du -hs / 2>/dev/null
3.6G	/

```

If you absolutely need writable ignite dir, you could either glue RUN commands together, or you may set up a bunch of symlinks:
```
/opt/apache-ignite-2.8.0/bin ... original archive
/opt/ignite-2.8.0/ - real directory
/opt/ignite-2.8.0/bin -> symlink to /opt/apache-ignite-2.8.0/bin
...
```
---
[1] - https://www.datawire.io/not-engineer-running-3-5gb-docker-images/
[2] - https://docs.docker.com/storage/storagedriver/

Reply:
@mshonichev, 
AFAIU, the question about the necessity of "RUN chmod a+wr /opt -R".
Seems, it can be safely removed since other folders used for write access.
Removed this.